webdriver-spec.html

```diff
@@ -4890,36 +4902,23 @@
  <li><p>If the <a>current browsing context</a> is <a>no longer open</a>,
   return <a>error</a> with <a>error code</a> <a>no such window</a>.
 
- <li><p>Let <var>element result</var> be the result of <a>getting a known element</a>
-  by <a>UUID</a> reference <var>element id</var>.
-
- <lI><p>Let <var>element</var> be <var>element result</var>’s data
-  if it is a <a>success</a>.
-  Otherwise return <var>element result</var>.
+ <li><p>Let <var>element</var> be the result of <a>trying</a>
+  to <a>get a known element</a> by <a>UUID</a> reference <var>element id</var>.
 
  <li><p>If the <var>element</var> is
   an <a><code>input</code> element</a> in the <a>file upload state</a>
   or it does not have a <a>container</a> <a>element</a>,
   return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
 
- <li><p><a>Scroll into view</a>
-  the <var><a>element</a></var>’s <a>container</a> <a>element</a>.
-
- <li><p>Wait in an implementation-specific way
-  up to the <a>session implicit wait timeout</a>
-  for the <a>container</a> to become <a>pointer-interactable</a>.
-
- <li><p>If <a>element from point</a> for
-  the <a>in-view centre point</a> of <var>element</var>
-  is not equal to the <var>element</var>’s <a>container</a>,
-  return <a>error</a> with <a>error code</a> <a>element click intercepted</a>.
+ <li><p><a>Scroll into view</a> <var>element</var>’s <a>container</a>
+  if it is not <a>pointer-interactable</a>.
 
- <li><p>If <var>element</var>’s <a>container</a>
-  is not <a>pointer-interactable</a>,
+ <li><p>If <var>element</var>’s <a>container</a> is still not <a>in view</a>,
```

Comment:
So I intentionally left this out because I didn’t understand what it did. Is the intention to make the command synchronous by waiting for it to finish scrolling before returning?

If so, I’m not sure we can wait for the full length of the duration since if the element is clearly still obscured after scrolling, we will just sit idle doing nothing.

Anyway, I can add it back in and raise a separate issue on this and we can discuss it there.